src/app/components/authorized/add-new-subreddit/add-new-subreddit.component.scss

```diff
@@ -1,6 +1,15 @@
-.typeButton:focus{
-    color: blue;
+@import '../../../../colors.scss';
+@import '../../../../mixins.scss';
+
+form {
+    @include form;
+}
+
+.gridWrapper {
+    display: grid;
```

Comment:
You are reusing gridWrapper and wrapper on multiple places. This actually may be reason to move it into root styles.